managed/services/inventory/agents.go

```diff
@@ -1311,6 +1314,15 @@ func (as *AgentsService) AddExternalExporter(ctx context.Context, p *inventoryv1
 		as.vmdb.RequestConfigurationUpdate()
 	}
 
+	// Schedule immediate status check after first scrape interval (15 seconds)
+	// This allows VictoriaMetrics to scrape the exporter and generate 'up' metric
+	agentID := agent.AgentId
+	go func() {
+		time.Sleep(15 * time.Second)
```

Comment:
IMO everything should be context-bound and the sleep itself should be cancellable.